.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+matrix:
+  include:
+  - python: 3.6
+    env: TOXENV=py36
```

Comment:
Could you please add Python 3.5? Scrapy still supports Python 3.5 (because Ubuntu 16.04 LTS ships with Python 3.5), so to make itemloaders Scrapy dependency it needs to support Python 3.5 as well.

Reply:
pypy3 would be nice, too. There is no need to install a custom version; Travis's pypy3 should work.